src/applications/appeals/testing/sc/utils/focus.js

```diff
@@ -18,7 +19,14 @@ export const focusEvidence = (_index, root) => {
   });
 };
 
-export const focusH3AfterAlert = () => {
-  scrollTo('topContentElement');
-  focusElement('h3#header');
+export const focusH3AfterAlert = ({ name, onReviewPage } = {}) => {
+  if (name && onReviewPage) {
+    // name, editing (alert only visible in edit mode), reviewEditFocusOnHeaders
+    const editing = true; // alert only visible in edit mode
+    const reviewEditFocusOnHeaders = true; // from form/config.js setting
+    focusReview(name, editing, reviewEditFocusOnHeaders);
+  } else {
+    scrollTo('topContentElement');
+    focusElement('h3#header');
+  }
```

Comment:
I hate that this is copy/pasta'd, I hate it more that the comments and implementation are inconsistent :-(

Reply:
DOH, I'll fix it!

Reply:
I copy/pasta'd the entire SC form into the prototype... once the research is done, I can then perform the arduous process of adding all these changes back into SC behind a feature toggle 🙀 - so keeping the code "nearly" the same may hopefully make it easier to find the changes.